ocaml/runtime/major_gc.c

```diff
@@ -1019,7 +1025,7 @@ void caml_major_collection_slice (intnat howmuch)
                    (intnat) (p * 1000000));
   caml_gc_message (0x40, "work backlog = %"
                          ARCH_INTNAT_PRINTF_FORMAT "du\n",
```

Comment:
```suggestion
                         ARCH_INTNAT_PRINTF_FORMAT "d\n",
```
This `u` was an ASCII representation of a mu for micro. Not needed anymore.